### DISCUSSION TITLE
Delta time

### DISCUSSION THREAD

**Discussion by simeonericcardo** [theOehrly/Fast-F1#592] [General]
It is in program a new version of the delta_time feature? The ones that is present now is not too accurate

**Comment by theOehrly** [comment]
I am aware that it is not very accurate. This is noted in the documentation and has been a problem for a long time. But it is not easy to fix because the source data is not very accurate in the first place. It might even be impossible to make it more accurate.
If you have ideas for improving it, feel free to share them. Any help is welcome.
